configure.ac

```diff
@@ -2012,7 +2012,7 @@ LIBS_TEMP="$LIBS"
 unset LIBS
 LIBS="$LIBS_TEMP"
 
-ac_configure_args="${ac_configure_args} --disable-shared --with-pic --enable-benchmark=no --enable-module-recovery --enable-module-schnorrsig"
```

Comment:
Should be then explicitly disable some others?
https://github.com/bitcoin-core/secp256k1/commit/41e8704b484652cf5bbb2b7ecc27feedc3cf0ae1

`--disable-module-ecdh` and `--disable-module-extrakeys`? Or were these already enabled indirectly?

Reply:
`schnorrsig` depends on `extrakeys`, so `extrakeys` is automatically enabled already now.

`ecdh` is a prerequisite for https://github.com/bitcoin/bitcoin/pull/23561/ (see https://github.com/bitcoin/bitcoin/pull/23561/commits/240fd59a4817df1b3375c03d03dc0cbc9550530d) but I'm not able to tell if this is a reason to enable it right now.